public-html/asset/scss/components/form.scss

```diff
@@ -108,12 +95,9 @@ button {
 }
 
 /* Darker background on mouse-over */
-.iconbtnedit:hover {
-	background-color: $blue;
-}
 
-.iconbtndelete:hover {
-	background-color: $red;
+.icon-btn-delete:hover {
+	background-color: $buttonhover !important;
```

Comment:
for use &:hover in next line like this
.icon-btn-delete {
   &:hover {
   }
}

Reply:
Done!